docs/available_commands.md

```diff
@@ -129,7 +133,7 @@ When passing the targets, names and values in a file, the file must conform to t
             "id": {
                 # Could be any one of the identifying properties in CycloneDX.
                 # Multiple identifiers are not allowed (with the special exception of name,
-                # group and version which are only valid together)
+                # group and version or version range which are only valid together)
```

Comment:
This makes it sound as if *version range* was a different field than *version* (which would arguably be a good idea but isn't the case).

Reply:
Changed since its more practical and the initial considerations against it are no longer valid.